app/javascript/css/main.scss

```diff
@@ -319,6 +319,11 @@ ul.list-right-section {
     background-color: var(--warning-status-tint);
     border: 1px solid var(--warning-border);
   }
+
```

Comment:
also need to add in an info badge. need to confirm with alex r, but something like:
`  &.badge-info {
    background-color: var(--info-tint);
    border: 1px solid var(--info-color);
  }`